engine/runtime/src/main/java/org/enso/interpreter/node/callable/resolver/HostMethodCallNode.java

```diff
@@ -76,9 +93,12 @@ public static PolyglotCallType getPolyglotCallType(
       return PolyglotCallType.READ_ARRAY_ELEMENT;
     } else if (library.isString(_this)) {
       return PolyglotCallType.CONVERT_TO_TEXT;
-    } else {
-      return PolyglotCallType.NOT_SUPPORTED;
+    } else if (library.isDate(_this)) {
+      if (!library.isTime(_this)) {
```

Comment:
We don't want `LocalDateTime` (returns `true` for both `isDate` as well as `isTime`) to be converted to `Date` (at least not as part of this PR).

Reply:
Surely wa want cases for `Time` and `Duration` here as well?

Reply:
was assuming we would add these using this PR as guide. Seemed like something we could take of Jaroslav's list

Reply:
I'd be glad if I could merge this PR, finally. Letting @jdunkerley , @hubertp , co. convert the `Time` and `Duration` would help me focus on investigating the language server protocol.